<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>412</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>File Intrusion</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/ProjectLegacy.png</normaloff>images/ProjectLegacy.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius:20px;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Add_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(6, 139, 255);
color:white;
border-radius:10px;
font:bold;
</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Remove_btn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(6, 139, 255);
color:white;
border-radius:10px;
font:bold;</string>
    </property>
    <property name="text">
     <string>REMOVE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Monitor_btn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color: rgb(12, 255, 32);
border-radius:10px;
font:bold;</string>
    </property>
    <property name="text">
     <string>MONITOR</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>118</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>3DS Fonticon</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="Info_label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-style:centre;
color:green;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Check_btn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(141, 84, 255);
color:white;
border-radius:10px;
font:bold;
</string>
    </property>
    <property name="text">
     <string>Check Files</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Checklist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
border-width:5px;
border-radius:2px
</string>
    </property>
   </widget>
   <widget class="QListWidget" name="mon_mode">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>231</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
border-color: rgb(255, 25, 5);
border-width:2px;
border-radius:2px;
color:red;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Files</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Breach</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Set_btn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>192</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
color:white;
border-radius:5px;
</string>
    </property>
    <property name="text">
     <string>SET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Monitoroff_btn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red;
color:white;
border-radius:10px;
font:bold;</string>
    </property>
    <property name="text">
     <string>MONITOR OFF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
